README.md

```diff
@@ -7,7 +7,14 @@
 [![devDependency Status](https://david-dm.org/hoodiehq/hoodie-app-tracker/dev-status.svg)](https://david-dm.org/hoodiehq/hoodie-app-tracker#info=devDependencies)
 
 Tracker is a simple Hoodie app, meant to be a starting point to play and build
-with Hoodie.
+with Hoodie. Find the repository on [GitHub](https://github.com/hoodiehq/hoodie-app-tracker).
+
+## Requirements
+
+Tracker needs `node >=4`. To see which version you have installed, run
```

Comment:
It could also be `hoodie-app-tracker`. I'm not sure, tbh – they all make sense to me, and the difference is rather small.

Breaking it down, `node` is a requirement for `hoodie` (plain: 'Hoodie') which is a requirement for `hoodie-app-tracker` (plain: 'Tracker').

I went with 'Tracker' because that's how the project ist referenced in the paragraph above.